mutt_commands.h

```diff
@@ -29,6 +29,7 @@ struct Buffer;
  * enum CommandError - Error codes for command_t parse functions
  */
 enum CommandError {
+  COMMAND_SUC =  0,
   COMMAND_ERR = -1,
   COMMAND_WAR = -2,
   COMMAND_FIN       ///< Distinguish "finish" command from successful return.
```

Comment:
Hmm...   What do you suppose the value of `COMMAND_FIN` is?!

Reply:
Oh sorry, the value was 1! Since it was only used in the finish command I didn't associate a value.

```suggestion
  COMMAND_FIN =  1  ///< Distinguish "finish" command from successful return.
```